edelphi/src/main/java/fi/metatavu/edelphi/pages/IndexPageController.java

```diff
@@ -90,10 +101,33 @@ public int compare(DelfoiBulletin o1, DelfoiBulletin o2) {
         return o2.getCreated().compareTo(o1.getCreated());
       }
     });
+
+    if (!ignoreBulletinLocale) {
+      TextLanguageDetector langdetect = null;
+      try {
+        langdetect = TextLanguageDetector.getInstance();
+      } catch (IOException e) {
+        e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)